lesson24/src/js/main.js

```diff
@@ -0,0 +1,48 @@
+const showModal = (target)  => {
+    target.classList.remove('hidden');
+    target.setAttribute('aria-modal', 'true');
+    target.setAttribute('role', 'dialog');
```

Comment:
Thank you for your comment!

Thank you for the detailed information about the accessibility implementation. 
I will look into it again and consider the implementation.

For adding attributes in JS, I made it by referring to the source code of TailwindCSS's UI component library flowbite.
But as you point out, it might be better to set it in HTML, so I'll reconsider.

https://github.com/themesberg/flowbite/blob/main/src/components/modal.js

Reply:
The aria attribute and required information is now set in the HTML.
https://github.com/nakagawamai/js-lesson/pull/25/commits/15e82850b8538cfd75fb13f31ee1b84434ec329b

Also, added processing of the focus control to meet accessibility requirements.
https://github.com/nakagawamai/js-lesson/pull/25/commits/1bf16cecca63808f2793f3a1d26408ba8928c077